### PR TITLE
Week 18 - Project Mongo API - Johanna Rexin

### PR DESCRIPTION
**Pull request by johannarexin**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -3,36 +3,72 @@ import bodyParser from 'body-parser'
...
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number
+})
```
Comment (ssundblom) [inline server.js:33]: Smart to save all the endpoints to the model, makes it much easier to add them later on! 👌🏽

File: server.js [hunk 2]
```diff
@@ -3,36 +3,72 @@ import bodyParser from 'body-parser'
...
+      const newArtist = new Artist(item)
+      newArtist.save()
+    })
+  }
+  seedDatabase()
+}
+
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('API of top songs on Spotify using Mondo DB!')
```
Comment (ssundblom) [inline server.js:48]: Super small typo here - Mongo DB ☺️🌟

**Review by ssundblom — COMMENTED** [review]
Good job Johanna 🌟 
I only seem to find one small typo, that doesn't even matter that much hahah! 

One thing that's not in requirements, but could be nice to have some sort of error message when searching for something that doesn’t exists in the database. 
 
Your project meets the requirements for blue level 💙

File: server.js [hunk 3]
```diff
@@ -3,36 +3,72 @@ import bodyParser from 'body-parser'
...
-const port = process.env.PORT || 8080
+const port = process.env.PORT || 8081
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+const Artist = mongoose.model('Artist', {
```
Comment (KarolCislowski) [inline server.js:18]: You do not store a list of artists here, so the Name of the model is not correct.  Please give it a fitting name like `songs` or `tracks`

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please change this model name, and it will be really great project.

File: server.js (hunk 3 above)
Comment (johannarexin) [reply server.js]: Hello! Thanks for the review :) Now I've done a new push with changed name of the model. /Johanna

**Review by KarolCislowski — APPROVED** [review]
Nice 😺

**Pull request closed without merging**